multi-node/aws/pkg/config/config.go

```diff
@@ -0,0 +1,326 @@
+package config
+
+//go:generate go run templates_gen.go
+//go:generate gofmt -w templates.go
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"strings"
+	"text/template"
+
+	"github.com/coreos/coreos-cloudinit/config/validate"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	credentialsDir = "credentials"
+	userDataDir    = "userdata"
+)
+
+func newDefaultCluster() *Cluster {
+	return &Cluster{
+		ClusterName:              "kubernetes",
+		ReleaseChannel:           "alpha",
+		VPCCIDR:                  "10.0.0.0/16",
+		InstanceCIDR:             "10.0.0.0/24",
+		ControllerIP:             "10.0.0.50",
+		PodCIDR:                  "10.2.0.0/16",
+		ServiceCIDR:              "10.3.0.0/24",
+		KubernetesServiceIP:      "10.3.0.1",
+		DNSServiceIP:             "10.3.0.10",
+		K8sVer:                   "v1.1.4",
```

Comment:
I know that using the coreos hyperkube images is still a TODO with this PR - but just noting here this version will need to be updated.